### PR TITLE
stable/kube2iam: added support for aws access keys

### PR DESCRIPTION
**Pull request by jescarri**
Added a secret resource to allow setting AWS access keys and region

### CODE REVIEW

**Comment by jescarri** [comment]
bumping CLA.

File: stable/kube2iam/templates/secret.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{{- if or (and .Values.aws.secret_key .Values.aws.access_key) -}}
```
Comment (mgoodness) [inline stable/kube2iam/templates/secret.yaml]: Delete the `or` and the parens.

File: stable/kube2iam/templates/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+metadata:
+  labels:
+    app: {{ template "kube2iam.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "kube2iam.fullname" . }}
+type: Opaque
+data:
+{{- if and .Values.aws.secret_key .Values.aws.access_key }}
```
Comment (mgoodness) [inline stable/kube2iam/templates/secret.yaml]: Don't need this conditional.

**Review by mgoodness — CHANGES_REQUESTED** [review]
Couple of changes, otherwise LGTM

**Review by mgoodness — APPROVED** [review]

**Pull request merged**